### PR TITLE
Update CI.yml

### PR DESCRIPTION
**Pull request by james-d-mitchell**

### CODE REVIEW

**Comment by james-d-mitchell** [comment]
@alex-konovalov @wilfwilson any ideas why the manual isn't building? It seems like the `aaa` package is not in a directory where GAP can find it, so line 25 in `makedoc.g` fails (the list returned by `PackageInfo("aaa");` is empty):

https://github.com/gap-packages/aaa/blob/154ed02603600272f3f1516cc402a1a61bf44658/makedoc.g#L25

**Comment by wilfwilson** [comment]
I'll have a quick look.

**Comment by wilfwilson** [comment]
Yes, it was failing because the `aaa` package could not located by GAP.

The bit of the `gap-actions` CI setup that teaches GAP about the location of the given package is, for whatever reason, currently in the Action `gap-actions/run-pkg-tests`.

Therefore GAP didn't know about its `PackageInfo` file, which is what the `makedoc.g` file was using to determine the directory containing the `aaa` package installation.

The convention of a `makedoc.g` file is that GAP should be started from the same directory that contains the `makedoc.g` file. Therefore, the location of the `makedoc.g` file = location of the package installation = `DirectoryCurrent()` = `.`, so the issue can be worked around.

For a thoroughly robust setup, I recommend seeing how it's done in the Digraphs package (in particular, that shows how to integrate a `makedoc.g` file with a `PkgMakeDoc()` function).

**Comment by wilfwilson** [comment]
I've pushed some commits that hopefully sort things out well enough for now. You can see how it's inspired by the Digraphs `makedoc.g` file.

**Comment by wilfwilson** [comment]
This was reported by @alex-konovalov in #10, which this PR now fixes.

**Comment by fingolfin** [comment]
AutoDoc also takes care of this, by the way

File: makedoc.g [hunk 1]
```diff
@@ -4,6 +4,14 @@
 # This file is a script which compiles the package manual.
 # To complile the doc for aaa run the command AaaMakeDoc(); in gap.
 
+if not IsDirectoryPath("gap")
+    or not "transducer.gd" in DirectoryContents("gap") then
```
Comment (fingolfin) [inline makedoc.g:8]: Could also do this:
```suggestion
if not IsExistingFile("gap/transducer.gd" then
```

**Review by fingolfin — APPROVED** [review]

**Comment by fingolfin** [comment]
Hopefully https://github.com/gap-actions/build-pkg-docs/pull/10 will help prevent similar trouble in the future

**Comment by le27** [comment]
@fingolfin @wilfwilson Thank you very much for the help

**Pull request merged**